src/GovUk.Education.ExploreEducationStatistics.Admin/Services/ReleaseDataFileService.cs

```diff
@@ -311,13 +292,17 @@ await _contentDbContext.Entry(dataFile)
 
                                                     var zipFile = await _releaseDataFileRepository.CreateZip(
                                                         filename: zipFormFile.FileName.ToLower(),
+                                                        contentLength: zipFormFile.Length,
+                                                        contentType: zipFormFile.ContentType,
                                                         releaseId: releaseId,
                                                         createdById: _userService.GetUserId());
 
                                                     var dataFile = await _releaseDataFileRepository.Create(
                                                         releaseId: releaseId,
                                                         subjectId: subjectId,
                                                         filename: archiveFile.DataFileName,
+                                                        contentLength: archiveFile.DataFileSize,
+                                                        contentType: "text/csv",
```

Comment:
During the files migration except for a handful of cases all the files got a `ContentType` that's not `application/octet-stream` so I would say this was never an issue in Prod or it got fixed a long time ago. I'll recommend that EES-1460 is retested or just closed off once this work is complete.

Setting the content type for non-zip data files using `dataFormFile.ContentType`, `metaFormFile.ContentType`, `upload.File.ContentType` (ancillary files), and `formFile.ContentType` (chart files)  seems to have been working.

I opened [https://dfedigital.atlassian.net/browse/EES-3572](https://dfedigital.atlassian.net/browse/EES-3572)  and did a bit of investigation into the ones with `application/octet-stream`, and fixed those up in Prod. They look like they are large zip file uploads where we probably uploaded them during support requests manually using storage explorer and never set the content type property.

Interestingly that work spotted one chart not displaying correctly in the Admin. I've corrected the ContentType on the File in the db now so hopefully it will be fixed when this PR gets deployed.

Reply:
That sounds good as I can't think of a reason why the data file or meta file shouldn't be `test/csv`. I've made this change now, thanks.